syft/frameworks/torch/tensors/interpreters/abstract.py

```diff
@@ -111,7 +111,8 @@ def wrap(self) -> torch.Tensor:
         wrapper = torch.Tensor()
         wrapper.child = self
         wrapper.is_wrapper = True
-        # wrapper.child.parent = weakref.ref(wrapper)
+        if self.id is None:
```

Comment:
Can we set something up in the constructor to make it so that self.id is never none?

Reply:
I've make a change in __ init__ of abstract tensor but it's not trivial in general as for torch tensor  the "register to init" function in hook.py sometimes doesn't run